releases-istio/apps/catalog-helm-release.yaml

```diff
@@ -45,6 +45,8 @@ spec:
         value: "mysql-svc.nordmart-dev-apps.svc.cluster.local"
       - name: MYSQL_DB_NAME
         value: "catalog"
+      - name: KAFKA_ADDRESS
+        value: "kafka-cluster-kafka-bootstrap.nordmart-dev-apps:9092"
```

Comment:
Sure this is the service name for kafka ?

Reply:
Yes, tested locally.